_includes/i18n-as-script.html

```diff
@@ -0,0 +1,11 @@
+{% comment %} get the current translations from Jekyll, to be used in scripts {% endcomment %}
+<script>
+  window.i18n = {
+    url_base: '{{url_base}}',
+    active_language: '{{vars_active_language.tag}}',
+    search_placeholder: '{{ i18n.search_placeholder }}',
+    search_by_algolia: '{{ i18n.search_by_algolia }}',
+    search_by_read_more: '{{ i18n.search_by_read_more }}',
```

Comment:
I suspect you should be JSON encoding these, as the translations in some languages may have apostrophes in them. We can follow up on that separately.